src/services/auth/plugins/mobile/mobile.test.ts

```diff
@@ -603,13 +580,15 @@ describe('Mobile Endpoints', () => {
 
       expect(responseRegister.statusCode).toBe(StatusCodes.NO_CONTENT);
       expect(mockSendEmail).toHaveBeenCalledTimes(1);
-      const authUrl = new URL(mockSendEmail.mock.calls[0][2]);
+
+      const tokenRegex = /\?t=([\w\-\.]{1,1000})/;
```

Comment:
Apparently this regex is also used in `password/index.test.ts` could we extract it into a test utils file ?